NMS-13446: provisiond docs updates (#3577)
OpenNMS/opennms: docs/modules/operation/pages/provisioning/policies/policies.adoc

[[policies]]
= Policies

The Policy API in Provisiond allows you to control the persistence of discovered IP and SNMP interface entities and node categories during the scan phase.

> Policies are defined by editing the foreign source definition (Edit Definition button) of a requisition.

Reviewer: ```suggestion
Edit the foreign source definition (Edit Definition button) of a requisition to define policies.
```

== Common policy parameters

The matchBehavior action is a boolean control that determines how the optional parameters will be evaluated.

> Setting this parameter’s value to `ALL_PARAMETERS` causes Provisiond to evaluate each optional parameter with boolean AND logic and the value `ANY_PARAMETERS` will cause OR logic to be applied.

Reviewer: ```suggestion
Setting this parameter’s value to `ALL_PARAMETERS` causes Provisiond to evaluate each optional parameter with boolean AND logic and the value `ANY_PARAMETERS` causes OR logic to be applied.
```
